docs/guides/chia-gaming/gaming-space-poker-rules.md

```diff
@@ -0,0 +1,91 @@
+---
+slug: /guides/gaming-space-poker-rules
+title: Space Poker Rules
+---
+
+:::warning Coming Soon
+
+Space Poker will be available in a future release. The rules below are based on current known game mechanics and may be subject to change.
+
+:::
+
+## Objective
+
+Space Poker is a poker variant played with two simultaneous games. Players receive two hole cards with a boost bit (+ or -) that affects hand rankings. The goal is to win both games or maximize winnings across both games.
+
+## Setup
+
+- Each player receives two hole cards selected at random from 2-A (T=10, J=11, Q=12, K=13, A=14)
+- Each hole card has a boost bit which is + or - with a ⅓ chance of being +
+- Both players ante a minraise at the start of each hand
+- All raise amounts are multiples of the minraise
+- Two games proceed simultaneously until both are complete
+
+## Hand Rankings
+
+From lowest to highest:
+
+1. **High Card**: No pair or better
+2. **Pair**: Two cards of the same rank
+3. **Two Pair**: Two different pairs
+4. **Set (Three of a Kind)**: Three cards of the same rank
+5. **Full House**: Three of a kind plus a pair
+6. **Straight**: Five consecutive cards (no ace-to-five straight)
+7. **Four of a Kind**: Four cards of the same rank
+8. **Five of a Kind**: Five cards of the same rank
+
+**Important Differences from Traditional Poker:**
+
+- **No Flushes**: Unlike traditional poker, there are no flushes in Space Poker
+- **Five of a Kind**: This is the highest possible hand
+- **No Ace-to-Five Straight**: The straight A-2-3-4-5 is not valid
+
+## Tiebreakers
+
+1. **Hand Type**: Higher hand type wins
+2. **Boost Bit**: The boost bit is the first tiebreaker after hand type
+3. **Standard Poker Tiebreaks**: After boost bit, standard poker tiebreaking rules apply
+
+## Gameplay
+
+### Turn Structure
+
+- Players always alternate turns
+- On your turn, you can either **call** or **raise**
```

Comment:
This is a bit confusing. Is call the same as check, and is raise the same as bet?

Reply:
Technically yes, instead of separating the terminology for each player I consolidated the terms:

Starting player - option to check or bet
Second player - option to call (can be considered a check) or raise

Also, how the current vibecoded version of the game looks the user does not have a check button and instead bets 0.

We can definitely rework this rules page as the game rules are finalized and coded